src/test/pages/Accordion.test.tsx

```diff
@@ -24,20 +24,39 @@ describe("AccordionItem", () => {
     expect(defaultProps.setOpenState).toHaveBeenCalledWith("1");
   });
 
-  it("applies the correct classes when open", () => {
+  it("renders the section content correctly when open", () => {
```

Comment:
another tip: if you're testing that it shows when it's `open` like this, make sure it's not visible when it's shut too. That means you know it's working properly as a test.

Reply:
Made wrapper component which uses the accordion item. Made a test to see if the accordion is closed and the sections are not in the document

Reply:
As discussed, added in a snapshot test